node/src/sub_lib/configurator.rs

```diff
@@ -6,11 +6,6 @@ use masq_lib::ui_gateway::NodeFromUiMessage;
 use std::fmt;
 use std::fmt::{Debug, Formatter};
 
-#[derive(Debug, actix::Message, Clone, PartialEq, Eq)]
```

Comment:
Put this code back in; it will be used by GH-728. Perhaps you could put a comment near it referring to GH-728 to make it easier to find for whoever picks up that card.

Reply:
Done! 👍